### PR TITLE
Use netty-tcnative version from netty-bom

### PR DESCRIPTION
**Pull request by kwart**
As we already import `netty` BOM in the `dependencyManagement`, we don't need to specify the `netty-tcnative` version which is covered.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1947,7 +1947,6 @@
             <dependency>
```
Comment (ldziedziul) [inline pom.xml:1947]: It's a `dependencyManagement` entry, since we have `netty-bom` here, I think this entry is redundant, WDYT?

Comment (kwart) [reply pom.xml:1947]: True! Nice catch I'll remove it.

**Comment by kwart** [comment]
run-lab-run

File: pom.xml (hunk 1 above)
Comment (TomaszGaweda) [reply pom.xml:1947]: True, because version of BOM is higher. If this version was higher, then it would make some sense :)

**Review by TomaszGaweda — APPROVED** [review]

**Comment by kwart** [comment]
Thanks, Tomasz.

**Pull request merged**